services/containers/helpers.go

```diff
@@ -0,0 +1,56 @@
+package containers
+
+import (
+	api "github.com/containerd/containerd/api/services/containers"
+	"github.com/containerd/containerd/containers"
+	"github.com/gogo/protobuf/types"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+func containersToProto(containers []containers.Container) []api.Container {
+	var containerspb []api.Container
+
+	for _, image := range containers {
+		containerspb = append(containerspb, containerToProto(&image))
+	}
+
+	return containerspb
+}
+
+func containerToProto(container *containers.Container) api.Container {
+	return api.Container{
+		ID:      container.ID,
+		Labels:  container.Labels,
+		Image:   container.Image,
+		Runtime: container.Runtime,
+		Spec: &types.Any{
+			TypeUrl: specs.Version,
```

Comment:
Wouldn't this be different per runtime and hence be taken from the container object somehow?